layouts/local-llm/single.html

```
@@ -0,0 +1,41 @@
+{{/* Local LLM Inference — WebGPU browser inference. */}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)